package.json

```diff
@@ -79,7 +79,7 @@
     "@hmcts/media-viewer": "2.10.1-RC.1",
     "@hmcts/nodejs-healthcheck": "1.7.0",
     "@hmcts/properties-volume": "0.0.13",
-    "@hmcts/rpx-xui-common-lib": "2.0.10",
+    "@hmcts/rpx-xui-common-lib": "2.0.16",
```

Comment:
`"govuk-frontend": "3.14.0",` also needs to be upgraded else the project will conflict and I believe use v3 instead